tests/tests_pytorch/helpers/datamodules.py

```diff
@@ -20,7 +20,7 @@
 from pytorch_lightning.core.datamodule import LightningDataModule
 from tests_pytorch.helpers.datasets import MNIST, SklearnDataset, TrialMNIST
 
-_SKLEARN_AVAILABLE = RequirementCache("sklearn")
+_SKLEARN_AVAILABLE = RequirementCache("scikit-learn")
```

Comment:
@carmocca It might be worth checking that this mistake was not made for other changes in #14475. 
My understanding is that RequirementCache is for packages, while we should use module_available for modules.

Reply:
Thanks for checking!